crates/secret-service-proto/src/v1/wire.rs

```diff
@@ -0,0 +1,244 @@
+use bitcoin::{
+    hashes::Hash,
+    taproot::{ControlBlock, TaprootError},
+    ScriptBuf, TapNodeHash,
+};
+use musig2::errors::{RoundContributionError, RoundFinalizeError};
+use rkyv::{with::Map, Archive, Deserialize, Serialize};
+use strata_bridge_primitives::scripts::taproot::TaprootWitness;
+
+use super::traits::{Musig2SessionId, SignerIdxOutOfBounds};
+
+#[derive(Debug, Clone, Archive, Serialize, Deserialize)]
+pub enum ServerMessage {
+    InvalidClientMessage,
+    OpaqueServerError,
+
+    OperatorSignPsbt {
+        sig: [u8; 64],
+    },
+    OperatorPubkey {
+        pubkey: [u8; 33],
+    },
+
+    SignP2P {
```

Comment:
I prefer explicitness
```suggestion
    P2PSignature {
```

Also why this is the only one that the context came before the object, e.g. `P2PPubkey` below and others.